examples/angular/src/pages/ui/primitive-poc/phone-number/phone-number.component.html

```diff
@@ -0,0 +1,22 @@
+<amplify-password-field
```

Comment:
These should be phone number fields

Reply:
Done.

Reply:
[075752a](https://github.com/aws-amplify/amplify-ui/pull/2420/commits/075752a0a6eb028fab66163ccc05ff6c0e625e94)